### PR TITLE
feat: create simple server

### PR DESCRIPTION
**Pull request by sebuzz**
closes: #22

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+			city: "Frankfurt",
+		},
+	],
+};
+app.get("/", (req, res) => {
+	res.send("Hello World");
+});
+
+app.get("/todos/", (request, response) => {
+	response.json(jason); // JSON.stringify(jason)
```
Comment (konrascher3) [inline server.js:25]: jason?

**Review by sedajune — APPROVED** [review]
Looks good, except for the little detail konrascher3 mentioned

File: server.js (hunk 1 above)
Comment (konrascher3) [reply server.js:25]: Ach, jason ist dein Array mit den Objekten drin, nvm...

Comment (sebuzz) [reply server.js:25]: Yes, it is just the name of the constant which contains the JSON Object

Comment (sebuzz) [reply server.js:25]: LGTM

**Review by konrascher3 — APPROVED** [review]

File: server.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+			name: "Florian",
+			city: "Frankfurt",
+		},
+	],
+};
+app.get("/", (req, res) => {
+	res.send("Hello World");
+});
+
+app.get("/todos/", (request, response) => {
```
Comment (marcbendixen) [inline server.js:24]: A better naming convention would be: `"/api/todos"` for this endpoint, because it's not a "normal route"

**Review by marcbendixen — COMMENTED** [review]
Added a small hint

**Pull request merged**